actioncable/test/javascript/src/test_helpers/consumer_test_helper.js

```diff
@@ -0,0 +1,59 @@
+import "../../vendor/mock-socket"
```

Comment:
Can we replace vendored [`mock-socket.js`](https://github.com/rails/rails/blob/14e49c2e28e2be4d97d132e99dd92ae9c4ddb4a9/actioncable/test/javascript/vendor/mock-socket.js) file with a `package.json` `devDependency` entry?

Reply:
👍 Replaced in https://github.com/rmacklin/rails/commit/6631c78a3745dc2e6941d77f810df047e81833e4